bower.json

```diff
@@ -5,41 +5,41 @@
   "authors": [
     "The Polymer Authors"
   ],
-  "private": true,
-  "homepage": "https://github.com/PolymerElements/app-layout",
+  "license": "https://polymer.github.io/LICENSE.txt",
+  "homepage": "https://elements.polymer-project.org/elements/app-layout",
```

Comment:
That's true: https://github.com/bower/spec/blob/master/json.md#homepage, but you wanted to be consistent with the other polymer elements. Otherwise, we would need to update them all :)